Formula/sigrok-cli.rb

```diff
@@ -0,0 +1,39 @@
+class SigrokCli < Formula
+  desc "Sigrok command-line interface to use logic analyzers and more"
+  homepage "https://sigrok.org/"
+  license "GPL-3.0-or-later"
+  revision 1
+
+  head "git://sigrok.org/sigrok-cli", branch: "master"
+
+  stable do
+    url "git://sigrok.org/sigrok-cli",
+        tag:      "sigrok-cli-0.7.2",
+        revision: "b584f959edb788f1731d5a304badf241ac21bf65"
+  end
+
+  livecheck do
+    url :head
+    regex(/^sigrok-cli-(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "glib"
+  depends_on "libsigrok"
+  depends_on "libsigrokdecode"
+
+  def install
+    system "./autogen.sh"
```

Comment:
Why do we need to regenerate this?

Reply:
They probably don't check in a `configure` script into their repo, and we're building from a Git clone. We should probably use a tarball if upstream have one, but they probably also have a broken `configure` script.

Reply:
We are pulling the sources from git, so all of the autoconf stuff needs to be generated.

Reply:
Is there no source tarball with a `configure` script?

Reply:
There is. We could switch to the source tar ball, but that would require different build steps for stable and head since the source tarball doesn't contain an autogen.sh script. It would also make build dependencies different for head and stable. In light of this, should we stick with the git tag or use the tar ball?

Reply:
Head builds aren't officially supported so I would urge you to just focus on the stable build at first.

Reply:
Differing dependencies and build steps between head and stable builds are also pretty typical.

Reply:
OK, switched to the tar balls for stable now. I _really_ want head builds to work for sigrok since the release cadence is insanely slow so I have made sure to test it with all of the three packages.